proposed/2021/TargetFrameworkAndPlatformsFilters.md

```diff
@@ -0,0 +1,210 @@
+# Target Framework and Platform Filters
+
+- [Jon Douglas](https://github.com/JonDouglas)
+- Start Date (2021-11-09)
+- [Package discovery: Show and search by TFM/package-compatibility #3098](https://github.com/NuGet/NuGetGallery/issues/3098)
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+When a developer is browsing for a NuGet package to include in their project, they can filter packages based on generations of .NET including the target framework & supported platforms that package targets.
+
+## Motivation
+
+<!-- Why are we doing this? What pain points does this solve? What is the expected outcome? -->
+Finding a package that is compatible with your project is a daily challenge for the average developer in the .NET ecosystem. Over 21% of developers fail to install a package today because of an incompatibility; it's one of NuGet's top errors known as `NU1202`. On top of it being one of the biggest error codes we see, it is also [the number #1 request on the `NuGetGallery` repository](https://github.com/NuGet/NuGetGallery/issues/3098) for good reason. It should be straight-forward for a developer to separate the generation of .NET they are currently targeting and find packages that are supported on the platforms that the respective generation supports.
+
```

Comment:
Given the problem statement in this section: 

> Finding a package that is compatible with your project is a daily challenge for the average developer in the .NET ecosystem. Over 21% of developers fail to install a package today because of an incompatibility; it's one of NuGet's top errors known as `NU1202`. 

Will showing the TFM badges in search result go a good distance towards solving the finding the "right" package?